machine-setup/README.md

```diff
@@ -0,0 +1,11 @@
+Steps to run the setupRancher.py and teardownRancher.py to setup and teardown machines for longhorn tests
+
+# first create a properties file to retrieve machine build results from container
+# it will contain CATTLE_TEST_URL and LONGHORN_BACKUP_SERVER_IP
+touch $PROPERTY_FILE_NAME
+
+# call setupRancher.py using environment variables to setup longhorn test machines
+docker run --rm -v $(pwd)/$PROPERTY_FILE_NAME:/$PROPERTY_FILE_NAME -e PROPERTY_FILE_NAME="$PROPERTY_FILE_NAME" -e GCE_SERVICE_KEY_JSON="$GCE_SERVICE_KEY_JSON" -e GCE_RANCHER_PROJECT_NAME="$GCE_RANCHER_PROJECT_NAME" -e GCE_RANCHER_PROJECT_ZONE="$GCE_RANCHER_PROJECT_ZONE" -e GCE_NFS_SERVER_NAME="$GCE_NFS_SERVER_NAME" -e GCE_RANCHER_SERVER_NAME="$GCE_RANCHER_SERVER_NAME" -e PACKET_HOST_NAMES="$PACKET_HOST_NAMES" -e PACKET_RANCHER_AUTH_TOKEN="$PACKET_RANCHER_AUTH_TOKEN" -e PACKET_RANCHER_PROJECT_ID="$PACKET_RANCHER_PROJECT_ID" -e PACKET_HOST_MACHINE_TYPE="$PACKET_HOST_MACHINE_TYPE" -e PACKET_HOST_OS_IMG="$PACKET_HOST_OS_IMG" -e PACKET_HOST_CLOUD_CONFIG="$PACKET_HOST_CLOUD_CONFIG" -e GCE_RANCHER_MACHINE_TYPE="$GCE_RANCHER_MACHINE_TYPE" -e GCE_RANCHER_OS_IMG="$GCE_RANCHER_OS_IMG" -e GCE_STARTUP_SCRIPT_RANCHER="$GCE_STARTUP_SCRIPT_RANCHER" -e GCE_STARTUP_SCRIPT_NFS="$GCE_STARTUP_SCRIPT_NFS" jimeng/longhorn-test:001 setupRancher.py
+
+# call teardownRancher.py using environment variables to destroy longhorn test machines
+docker run --rm -e GCE_SERVICE_KEY_JSON="$GCE_SERVICE_KEY_JSON" -e GCE_RANCHER_PROJECT_NAME="$GCE_RANCHER_PROJECT_NAME" -e GCE_RANCHER_PROJECT_ZONE="$GCE_RANCHER_PROJECT_ZONE" -e GCE_NFS_SERVER_NAME="$GCE_NFS_SERVER_NAME" -e GCE_RANCHER_SERVER_NAME="$GCE_RANCHER_SERVER_NAME" -e PACKET_HOST_NAMES="$PACKET_HOST_NAMES" -e PACKET_RANCHER_AUTH_TOKEN="$PACKET_RANCHER_AUTH_TOKEN" -e PACKET_RANCHER_PROJECT_ID="$PACKET_RANCHER_PROJECT_ID" jimeng/longhorn-test:001 teardownRancher.py 
```

Comment:
I think it's better to have a Makefile. Just let it run `flake8` first then `docker build` to build the image. Then we can generate the image by just run `make`. `docker push` need to be done manually though.